### DISCUSSION TITLE
差异化策略 - DaoCloud Enterprise

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#1079] [Announcements]
# 差异化策略 - DaoCloud Enterprise

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/kairship/policy/override-policy/

**Comment by liyuerich** [comment]
进入某一个多云实例后，在左侧导航栏中，点击策略管理 -> 部署策略，点击右上角的 YAML 创建按钮。

应该是 差异化策略

**Comment by ModetaNiu** [comment]
Already fixed in #1113
